UT4MasterServer/Services/MatchmakingService.cs

```diff
@@ -72,31 +80,13 @@ public async Task<List<GameServer>> ListAsync(GameServerFilter inputFilter)
 		if (inputFilter.BuildUniqueId != null)
 			doc.Add(new BsonElement(nameof(GameServer.BuildUniqueID), inputFilter.BuildUniqueId));
```

Comment:
You should avoid adding statement without braces in C#. Statement block without braces is more of JS thing.
You could leave only short statement without braces, like return/continue/break.